week-1-homework-Sila

bsilakaymak opened the pull request.

paulbremer commented on week1/code/project/components/CityWeather.js:
```diff
@@ -0,0 +1,20 @@
+            <div style = {{border:'2px solid black', margin:'10px', paddingLeft: '10px'}}> 
+                <CityName city = {city}/>
+                <Description city = {city}/>
+                <Details city = {city} kelvinToCelcius = {(kelvin)=>{return (kelvin-273).toFixed(2)}}/>
```
I'm pretty sure the API has an option to return Celcius! No need to calculate it if you can use the API for that!

paulbremer commented on week1/exercises-week1/src/components/Guarantee.js:
```diff
@@ -0,0 +1,44 @@
+  ]
+
+  return guaranteeData.map((guaranteeItem) =>{
+     return (<div className='guarantee' style = {{width:'33%', display:'inline-block'}}>
```
A width of 33% works but I know you learned about flexbox, try using that the next time!

paulbremer commented on week1/exercises-week1/src/components/Img.js:
```diff
@@ -0,0 +1,12 @@
+
+const Img = ({img, alt}) => {
+ return(
+     <img src = {img} alt={alt} ></img>
```
The image could be a self closing tag; `<img ... />`

bsilakaymak commented:
Because I had a problem with deploying exercises, I created a separate repo for exercises and I added the files that would be evaluated but apparently I forgot to add App.js file. I added the missing file and did minor changes accordingly. Thanks for your time
